resource/lookup_pipeline.yaml

```diff
@@ -0,0 +1,46 @@
+---
+jobs:
+- name: deploy
+  plan:
+
+  - get: release
+    trigger: true
+    params:
+      include_source_tarball: true
+
+  - get: common-tasks
+
+  - task: lookup-source
+    file: common-tasks/extract-release-tarball.yaml
+    output_mapping: {extracted: lookup-source}
+
+  - task: run-lookup-etl
+    config:
+      platform: linux
+      image_resource:
+        type: docker-image
+        source:
+          repository: quay.io/mojanalytics/lookup-tables-etl
+          tag: v0.0.1
+      run:
```

Comment:
if you add inputs before this line, eg:
```
image_resource:
  ...
inputs:
  - name: lookup-source
run:
  ...
```
Then you should have a `lookup-source` directory mounted in the working directory of your `lookup-tables-etl` container - IIRC